system-variables.md

```diff
@@ -2068,6 +2068,28 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - Default value: `ON`
 - This variable controls whether to enable GOGC Tuner.
 
+### tidb_gogc_tuner_max_value <span class="version-mark">New in v7.5.0</span>
```

Comment:
To maintain consistency, please ensure all system variables are listed in alphabetical order. Specifically, position `tidb_gogc_tuner_max_value` and `tidb_gogc_tuner_min_value` ahead of `tidb_gogc_tuner_threshold`.